lib/tasks/users.rake

```diff
@@ -33,4 +33,29 @@ namespace :users do
     remaining = User.where.not(latitude: nil, longitude: nil).where(state: nil).count
     p "#{remaining} users left to be updated.  Task can be reran in 24 hours."
   end
+
+  desc "Tags select users as Community Leaders"
+  task seed_leader: :environment do
+    users = [ {id: 4, first_name: 'Nell', last_name: 'Shamrell-Harrington' }, {id: 3, first_name: 'Rick', last_name: 'Rein'}]
+    users_id = User.where(id: [3,4])
+    users_id.each do |user|
+      user.tag_list.add(User::LEADER)
```

Comment:
Before you add the new tag to their list, you can check if they already have it present in their list.  That way you won't duplicate tags.  

For this you can use our [has_tag](https://github.com/OperationCode/operationcode_backend/blob/master/app/models/user.rb#L201-L203) method:

```ruby
next if user.has_tag?(User::LEADER)
user.tag_list.add(User::LEADER)
```